examples/sampling/graphbolt/link_prediction.py

```diff
@@ -311,7 +313,8 @@ def parse_args():
     parser.add_argument("--epochs", type=int, default=10)
     parser.add_argument("--lr", type=float, default=0.0005)
     parser.add_argument("--neg-ratio", type=int, default=1)
-    parser.add_argument("--batch-size", type=int, default=512)
+    parser.add_argument("--train-batch-size", type=int, default=512)
+    parser.add_argument("--eval-batch-size", type=int, default=2)
```

Comment:
Why eval batch so small

Reply:
Since the `neg_ratio` in evaluation stage is too large(1000), bigger eval batch_size will cause GPU OOM.